x/permission/keeper/keeper.go

```diff
@@ -570,3 +598,27 @@ func (k Keeper) RemoveExpiredPolicies(ctx sdk.Context) {
 		}
 	}
 }
+
+func (k Keeper) MigrateAccountPolicyForResources(ctx sdk.Context) {
```

Comment:
We'd better write some unit tests for the new method here and in `keys.go`

Reply:
added